conans/test/functional/toolchains/cmake/test_cmake.py

```diff
@@ -396,7 +394,7 @@ def test_toolchain_linux(self, build_type, cppstd, arch, libcxx, shared):
                 "CMAKE_SHARED_LINKER_FLAGS": arch_str,
                 "CMAKE_EXE_LINKER_FLAGS": arch_str,
                 "COMPILE_DEFINITIONS": defines,
-                "CMAKE_POSITION_INDEPENDENT_CODE": "ON"
+                "CMAKE_POSITION_INDEPENDENT_CODE": "ON" if not shared else ""
```

Comment:
I'd add a comment why this is conditional now

Reply:
done